src/screens/SendConfirmationSheet.js

```diff
@@ -0,0 +1,386 @@
+
+  const handlePress = useCallback(() => {
+    onPress({ checked: !checked, id, label });
+  }, [checked, id, label, onPress]);
+  return (
+    <Column marginBottom={20}>
+      <ButtonPressAnimation onPress={handlePress}>
+        <Row>
+          <Text
+            color={(checked && activeColor) || colors.blueGreyDark80}
+            size="lmedium"
+            weight="700"
+          >
+            {checked ? '􀃳 ' : '􀂒 '}
+          </Text>
+          <Text
+            color={(checked && activeColor) || colors.blueGreyDark80}
+            size="lmedium"
+            weight="700"
+          >
+            {label}
+          </Text>
+        </Row>
+      </ButtonPressAnimation>
+    </Column>
+  );
+};
+
+const defaultContactItem = randomColor => ({
+  address: '',
+  color: randomColor,
+  nickname: '',
+});
+
+export default function SendConfirmationSheet() {
+  const { colors, isDarkMode } = useTheme();
+  const { nativeCurrency } = useAccountSettings();
+  const { goBack, navigate, setParams } = useNavigation();
+  const { height: deviceHeight, isSmallPhone, isTinyPhone } = useDimensions();
+  const [isAuthorizing, setIsAuthorizing] = useState(false);
+  const insets = useSafeArea();
+  const { contacts } = useContacts();
+
+  const {
+    params: { asset, amountDetails, callback, isNft, network, to, toAddress },
+  } = useRoute();
+
+  const [alreadySentTransactions, setAlreadySentTransactions] = useState(0);
+
+  const { transactions } = useAccountTransactions(true, true);
+  const { userAccounts } = useUserAccounts();
+  const isSendingToUserAccount = useMemo(() => {
+    const found = userAccounts?.find(account => {
+      return toLower(account.address) === toLower(toAddress);
+    });
+    return !!found;
+  }, [toAddress, userAccounts]);
+
+  useEffect(() => {
+    if (!isSendingToUserAccount) {
+      let sends = 0;
+      transactions.forEach(tx => {
+        if (toLower(tx.to) === toLower(toAddress)) {
+          sends++;
+        }
+      });
+      if (sends > 0) {
+        setAlreadySentTransactions(sends);
+      }
+    }
+  }, [
+    isSendingToUserAccount,
+    setAlreadySentTransactions,
+    toAddress,
+    transactions,
+  ]);
+
+  const contact = useMemo(() => {
+    return get(
+      contacts,
+      `${[toLower(to)]}`,
+      defaultContactItem(getRandomColor())
+    );
+  }, [contacts, to]);
+
+  const [checkboxes, setCheckboxes] = useState([
+    { checked: false, label: 'I’m not sending to an exchange' },
+    {
+      checked: false,
+      label: `The person I’m sending to has a wallet that supports ${capitalize(
+        network
+      )}`,
+    },
+  ]);
+
+  const handleCheckbox = useCallback(
+    checkbox => {
+      const newCheckboxesState = [...checkboxes];
+      newCheckboxesState[checkbox.id] = checkbox;
+      setCheckboxes(newCheckboxesState);
+    },
+    [checkboxes]
+  );
+
+  const handleL2ExplainerPress = useCallback(() => {
+    navigate(Routes.EXPLAIN_SHEET, {
+      network,
```

Comment:
`network` not needed here currently (not used in route params in explain sheet, only the type is used)